.huskyrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./dist/config/huskyrc')
```

Comment:
Let's not include this one. Only one way to do the same thing.

Reply:
Alternatively we could keep this and have it require the source file instead, but just not expose it in the package. That way we don't have to repeat ourselves and people get only one way to do the same thing